util/topsql/reporter/reporter_test.go

```diff
@@ -302,8 +307,8 @@ func TestCollectCapacity(t *testing.T) {
 	collectedData := make(map[string]*dataPoints)
 	tsr.doCollect(collectedData, 1, genRecord(20000))
 	require.Equal(t, 5001, len(collectedData))
-	require.Equal(t, int64(5000), tsr.sqlMapLength.Load())
-	require.Equal(t, int64(5000), tsr.planMapLength.Load())
+	require.Equal(t, int64(20000), tsr.sqlMapLength.Load())
+	require.Equal(t, int64(20000), tsr.planMapLength.Load())
```

Comment:
why 20000

Reply:
Because the total SQL/Plan meta size is 20000, since now all SQL/Plan meta will be reported.